app/people/king.py

```diff
@@ -0,0 +1,5 @@
+class King:
+    def __init__(self, name: str, gold: int) -> None:
+        self.name = name
+        self.gold = gold
+        print(f"King {name} was born. He has {gold} gold coins.")
```

Comment:
The print statement uses a mix of single and double quotes. According to the checklist, you should use double quotes consistently throughout the code.